src/styles/StudentStyles/index.less

```diff
@@ -104,6 +104,10 @@
     border-radius: 25px;
 }
 
+.tasks-completed div {
+   // padding: 0% 10%;
+}
```

Comment:
can this class be removed since styling is commented out?

Reply:
Deleted!